lib/tasks/argo_testing.rake

```diff
@@ -61,8 +61,8 @@ if ['test', 'development'].include? Rails.env
         i = 0
 
         require 'webmock'
-        # only allow connections to Fcrepo and Solr
-        WebMock.disable_net_connect!(allow: ['localhost:8983', 'localhost:8984'])
+        # only allow connections to Fcrepo, Solr and workflow
+        WebMock.disable_net_connect!(allow: ['workflow:3000', 'solr:8983', 'fcrepo:8080', 'localhost:3000', 'localhost:8983', 'localhost:8984'])
```

Comment:
why localhost 3000?

Reply:
you are correct - not needed, and removed